kancolle_auto.sikuli/kancolle_auto.py

```diff
@@ -367,16 +371,7 @@ def init():
         combat_item = combat_module.Combat(global_regions['game'], settings)
         default_quest_mode = 'sortie'
         log_success("Combat module started (Sortie mode)")
-    if settings['pvp_enabled'] and settings['combat_enabled']:
-        if settings['pvp_fleetcomp'] == 0 or settings['combat_fleetcomp'] == 0:
```

Comment:
As far as I have been using as it is, the script doesn't switch fleets if the list doesnt specify more than 1 fleet, itll just ignore its value, see lines 232-233. So the end result is that the script behaves as before the patch if thats the case. 

Though I do agree that it will need some input checking and see that the values of ```FleetComps``` are either 1. Empty for disabling it and 2. that any value specified if not empty is from 1-10.